### PR TITLE
Module 1

### PR DESCRIPTION
**Pull request by keserukristof**

### CODE REVIEW

File: module-1/classification.js [hunk 1]
```diff
@@ -21,8 +21,21 @@ function grade(score) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    if (score < 0 || score > 100) {
```
Comment (horvathmilan) [inline module-1/classification.js]: I would suggest to try to solve the function with the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

**Review by horvathmilan — CHANGES_REQUESTED** [review]

File: module-1/classification.js (hunk 1 above)
Comment (keserukristof) [reply module-1/classification.js]: I rewrote that code, based on your suggestion.

Comment (keserukristof) [reply module-1/classification.js]: I committed and pushed the changes, based our last meeting (classification.js, fibonacci.js).

File: module-1/fibonacci.js [hunk 2]
```diff
@@ -15,18 +15,24 @@ function fibonacci(n) {
...
+    //     let sum = num1 + num2;
+    //     num1 = num2;
+    //     num2 = sum;
+    // }
+    // nThFibonacci = num2;
     // ...AND THIS COMMENT LINE!
-    return nThFibonacci;
+    //return nThFibonacci;
 }
+console.log(fibonacci("asd"));
```
Comment (horvathmilan) [inline module-1/fibonacci.js]: You can delete this line

**Review by horvathmilan — CHANGES_REQUESTED** [review]

**Review by horvathmilan — APPROVED** [review]

**Pull request merged**